packages/x-components/static-docs/experience-search-and-discovery/recommendations.md

```diff
@@ -10,40 +10,89 @@ tags:
   - popular
 ---
 
-The Recommendations UI component shows the most clicked products on the SERP, based on shopper
-interaction within a defined period. They help your shoppers to explore your product catalogue,
-guiding them to specific products without the need to launch any queries.
+Product recommendations, such as Top Products, Brand Recommendations, Next Products, and Vectorized Recommendations, help your shoppers explore your product catalogue, guiding them to specific products without the need to launch any queries.
 
-![Recommendations](~@assets/x/interface/x-recommendations.gif)
+<!-- ![Recommendations](~@assets/x/interface/x-recommendations.gif) -->
+![Recommendations](~@assets/x/interface/x-recommendations.svg)
 
-::: warning
+::: interact  
+Can't quite capture the concept? Learn more about
+[Recommendations](/explore-empathy-platform/features/recommendations-overview.md).
+:::
 
-Recommendations are generated using collective shopper behavior, generating a feed with the most
-clicked products. For a correct performance, make sure that your current search service supports
-this type of feature.
+## Product recommendation modules and components  
 
-:::
+The Interface X Components library includes the following modules and components to layout multiple experiences to show product recommendations in your search UI:
 
-::: interact
+<CardCarousel :cards="[
+    'ui_ref/recommendations',
+    'ui_ref/semantics',
+    'ui_ref/preview',
+    'ui_ref/nq'
+    ]"
+/>
 
-Can't quite capture the concept? Learn more about
-[Recommendations](/explore-empathy-platform/features/recommendations-overview.md).
+::: warning
+
+For correct performance, make sure that your current search service supports
+this type of features.
 
 :::
 
 ## Tailor the web experience
+Based on your business strategy and your shoppers’ needs, the Interface X Components library offers the following configuration options and properties:
 
-- Configure the position and place it wherever you prefer, although they usually appear on the
-  results page.
-- Show as many recommendations as you want.
+::: details Top Products UI experience
+Use the Recommendations module to configure the layout of Top Products in your commerce search UI. You can:
+- Configure the position and place it wherever you prefer, although it commonly appears on the SERP, among the product results.
+- Show as many product recommendations as you want.
 - Animate the display of the component at your ease.
 - Customize content. Show whatever you need: text, images, icons.
-- Extend the performance with results-related components.
+- Extend the performance with result-related components.
+
+:::
+
+
+::: details Vectorized Recommendations UI experience    
+
+Use the Semantics Queries module to configure the layout of Vectorized Recommendations in your commerce search UI. You can:
+- Configure the position and place it wherever you prefer, although it usually appears on the SERP, among the product results.
```

Comment:
As we will start using it in the No results scenario, I would include here a sentence clarifying that there are these two possibilities, use it in the Grid as Query Expansion, or suit in the No Result layout, that makes sens to you?